.vscode/launch.json

```diff
@@ -0,0 +1,14 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Program",
+            "program": "${workspaceFolder}\\src\\index.js"
+        }
+    ]
+}
```

Comment:
Hey, I think you commited this file by mistake :) would you please remove it from this patch?

Maybe you can should add the file `.vscode` to your local file `~/.gitignore`.

Reply:
DONE